templates/web/base/common_header_tags.html

```diff
@@ -1,7 +1,11 @@
 [% SET start = c.config.ADMIN_BASE_URL IF admin %]
 
 <meta http-equiv="content-type" content="text/html; charset=utf-8">
-<link rel="manifest" href="/.well-known/manifest.webmanifest">
+[% IF c.req.uri.path == '/waste' %]
```

Comment:
I think we want this to be on any waste page, not just the 'home' one - so could this check if the path starts with /waste instead